src/styles/blocks/on-view.scss

```diff
@@ -0,0 +1,163 @@
+.on-view {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__title {
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 32px;
+    letter-spacing: -1.5px;
+    text-align: center;
+    color: #161616;
+
+    @media (min-width: 1024px) {
```

Comment:
You should have three different designs, one for each screen size, as shown in the [Figma](https://www.figma.com/design/lSR1m42L9YwzQwzzxKwHpw/THE-MET?node-id=8590-29&node-type=canvas&t=HmJnyC86kTETP1FQ-0)

Your layout is not applied to `tablet view size`.

Also, it would be nice if you used more the `mixins` and `@include` to apply the styles for different view sizes, instead of using `@media`